docs/source/users/index.md

```diff
@@ -342,14 +344,29 @@ providers, as defined in [`langchain.llms`](https://langchain.readthedocs.io/en/
 - `openai-chat`
 - `sagemaker-endpoint`
 
-You can find a list of supported providers and models by running `%ai list`. Some providers
-define a list of supported models. If a provider does not define a list of supported models,
-consult the vendor's documentation. The [Hugging Face web site](https://huggingface.co/)
-includes a list of models, for example.
+:::{warning}
+As of v0.8.0, only the `%%ai` *cell* magic may be used to invoke a language
+model, while the `%ai` *line* magic is reserved for invoking subcommands.
+:::
+
+### Listing available models
+
+Jupyter AI also includes multiple subcommands, which may be invoked via the
+`%ai` *line* magic. Jupyter AI uses subcommands to provide additional utilities
+in notebooks while keeping the same concise syntax for invoking a language model.
+
+The `%ai list` subcommand prints a list of available providers and models. Some
+providers explicitly define a list of supported models in their API. However,
+other providers, like HuggingFace Hub, lack a well-defined list of available
+models. In such cases, it's best to consult the provider's upstream
+documentation. The [HuggingFace website](https://huggingface.co/) includes a
+list of models, for example.
+
+Optionally, you can specify a provider ID as a positional argument to `%ai list`
+to get all models provided by one provider. For example, `%ai list openai` will
+display only models provided by the `openai` provider.
 
-Optionally, you can pass a provider ID as a parameter to `%ai list` to get all
-models provided by one provider. For example, `%ai list openai` will display only models
-provided by the `openai` provider.
+### Abbreviated syntax
```

Comment:
Just to confirm, is this abbreviated syntax still supported? I wasn't sure about it before.

Reply:
Yes.